aries-site/src/examples/templates/drill-down-navigation/index.js

```diff
@@ -0,0 +1,3 @@
+export * from './drill-down-card-example';
+export * from './drill-down-datatable-example';
+export * from './drill-down-list-example';
```

Comment:
```suggestion
export * from './drill-down-list-example';

```
need new line at end of file